### ISSUE TITLE
remove ".com" from he repo name

### ISSUE THREAD

**Issue by DaveEveritt** [camping/whywentcamping.com#6]
"whywentcamping.com" now opens Apple Music - suggest renaming the repo by removing the ".com" to prevent accidental visits to a scam site

**Comment by chiperific** [comment]
Somebody else has the domain for sure. It directs me to this URL which wants me to install a Chrome extension: https://www1.securysearchapps.com/view/item_78256.html

**Comment by DaveEveritt** [comment]
> Somebody else has the domain for sure. It directs me to this URL which wants me to install a Chrome extension: https://www1.securysearchapps.com/view/item_78256.html

Indeed! We're discussing how to fix the various website issues in [this issue](https://github.com/camping/camping/issues/97)
